spring-boot-autoconfigure/src/main/java/org/springframework/boot/autoconfigure/cassandra/CassandraProperties.java

```diff
@@ -117,6 +117,53 @@
 	 */
 	private boolean ssl = false;
 
+	/**
+	 * Socket option: TCP keepalive.
```

Comment:
This field documentation is used for the description in the meta-data. We have a few guidelines (not start with `The`, end with a period, etc). Those "Socket option" and "Pooling option" looks noise to me. Or at least they should be rephrased. Field meta-data may be multi sentences but the first one should be short and as complete as possible.